src/test/unit_tests/jest_tests/test_nc_bucket_cli.test.js

```diff
@@ -242,6 +249,19 @@ describe('manage nsfs cli bucket flow', () => {
             await assert_bucket(bucket, bucket_options, config_fs);
             expect(bucket.s3_policy).toStrictEqual(bucket_policy);
         });
+
+        it('cli create bucket - with bucket name suffix `.json`', async () => {
+            await fs_utils.create_fresh_path(bucket_json_storage_path);
+            await fs_utils.file_must_exist(bucket_json_storage_path);
+            const action = ACTIONS.ADD;
+            const bucket_options = { config_root, ...bucket_suffix_json };
+            await exec_manage_cli(TYPES.BUCKET, action, bucket_options);
+            const bucket = await config_fs.get_bucket_by_name(bucket_suffix_json.name);
+            expect(bucket).not.toBeNull();
+            expect(bucket.name).toBe(bucket_options.name);
+            const res = await exec_manage_cli(TYPES.BUCKET, ACTIONS.STATUS, { name: bucket_suffix_json.name, config_root });
+            expect(JSON.parse(res).response.reply.name).toBe(bucket_options.name);
```

Comment:
To be aligned with the issue steps you can also test it using the `ACTIONS.LIST`.